pubspec.yaml

```diff
@@ -14,12 +14,8 @@ dependencies:
   flutter_localizations:
     sdk: flutter
   intl: ^0.17.0
-  local_storage_wishes_api:
-    path: packages/local_storage_wishes_api
-  wishes_api:
-    path: packages/wishes_api
-  wishes_repository:
-    path: packages/wishes_repository
+  wish_deposit:
```

Comment:
can we add a ci workflow for this package?

Reply:
Depends, what do you exactly mean with that 😄

Reply:
haha can we create a github workflow in the `.github/workflows/wish_deposit.yaml` to use `very_good_workflows` so that we run the standard checks whenever changes are made to `package:wish_deposit`?